### PR TITLE
added support for generating import api sdk

### PR DESCRIPTION
**Pull request by alex-ct**

### CODE REVIEW

File: languages/javalang/builder-renderer/java-builder-client/src/test/kotlin/io/vrap/codegen/languages/javalang/client/builder/BuilderTestCodeGenerator.kt [hunk 1]
```diff
@@ -70,6 +74,15 @@ class BuilderTestCodeGenerator {
         generatorComponent.generateFiles()
     }
 
+    @Ignore
```
Comment (acbeni) [inline languages/javalang/builder-renderer/java-builder-client/src/test/kotlin/io/vrap/codegen/languages/javalang/client/builder/BuilderTestCodeGenerator.kt]: Why did you add `@Ignore` here ?

Comment (alex-ct) [reply languages/javalang/builder-renderer/java-builder-client/src/test/kotlin/io/vrap/codegen/languages/javalang/client/builder/BuilderTestCodeGenerator.kt]: Honestly I dont even remember anymore, all of the tests in this class are ignored.

Comment (alex-ct) [reply languages/javalang/builder-renderer/java-builder-client/src/test/kotlin/io/vrap/codegen/languages/javalang/client/builder/BuilderTestCodeGenerator.kt]: There was some problem when running these tests.
